PartTwo/PartTwo/CustomView.m

```diff
@@ -0,0 +1,42 @@
+//
+//  CustomView.m
+//  PartTwo
+//
+//  Created by Daniel Distant on 9/18/15.
+//  Copyright © 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "CustomView.h"
+
+@implementation CustomView
+
+-(void)socialMediaViewDidTapLikeButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapLikeButton:view];
+}
+
+-(void)socialMediaViewDidTapCommentButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapCommentButton:view];
+}
+
+-(void)socialMediaViewDidTapShareButton:(CustomView *)view {
+    [self.delegate socialMediaViewDidTapShareButton:view];
+}
+
+- (IBAction)likeButtonTapped:(id)sender {
+    CustomView *view = [[CustomView alloc] init];
+    [self socialMediaViewDidTapLikeButton:view];
```

Comment:
There's no need call another method on self if that other method is only one line of code. You're not making it easier on yourself. Just go ahead and call the delegate directly in this method.